app/code/Magento/Config/etc/system.xsd

```diff
@@ -95,6 +95,7 @@
                     <xs:element name="header_css" type="xs:string" />
                     <xs:element name="resource" type="typeAclResourceId" />
                     <xs:element ref="group" />
+                    <xs:element name="frontend_model" type="typeModel" />
```

Comment:
I guess no. It will add the attribute for <group> node.
But I've added new element/node inside <section> node. It is used in this function
\Magento\Config\Block\System\Config\Edit::_prepareLayout()
`
    protected function _prepareLayout()
    {
        /** @var $section \Magento\Config\Model\Config\Structure\Element\Section */
        $section = $this->_configStructure->getElement($this->getRequest()->getParam('section'));
        $this->_formBlockName = $section->getFrontendModel();
        if (empty($this->_formBlockName)) {
            $this->_formBlockName = self::DEFAULT_SECTION_BLOCK;
        }
        ...
    }
`